docs/keto/guides/userset-rewrites.mdx

```diff
@@ -114,6 +117,42 @@ $ keto check User:Henning edit File private
 Allowed
 ```
 
+## (Optional) Local Setup and Configuration
+
+If you do not want to use the Ory Console you need to [install Keto](../install.mdx) locally. Next, create a `keto.yaml` file with
```

Comment:
> If you do not want to use the Ory Console

I think it's also about not using Ory Permissions but instead relying on a self-hosted instance, right?

Reply:
Yes.